.editorconfig

```diff
@@ -2,12 +2,15 @@ root = true
 
 [*]
 indent_size = 2
+indent_style = space
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 end_of_line = lf
 
 [*.{kt, kts}]
 ij_kotlin_imports_layout = *,java.**,javax.**,kotlin.**,kotlinx.**,^
-
-ktlint_standard_argument-list-wrapping = disabled
\ No newline at end of file
+ij_kotlin_allow_trailing_comma = false
+ij_kotlin_allow_trailing_comma_on_call_site = false
+ktlint_standard_argument-list-wrapping = disabled
+ktlint_function_naming_ignore_when_annotated_with=Composable
```

Comment:
```suggestion
ktlint_function_naming_ignore_when_annotated_with = Composable
```
for consistency

Reply:
wanted to, slipped in, thanks for noticing